packages/experiments/src/demo/Customizations.ts

```diff
@@ -0,0 +1,158 @@
+import { createTheme, ITheme } from 'office-ui-fabric-react/lib/Styling';
```

Comment:
File name is capitalized here vs the other version of this file which is lower cased.

Reply:
Don't we usually only capitalize filenames for a file that exports just a single class or interface?

Reply:
Sounds like a good standard, we should document that with import behavior 😄